source/modules/configuration/notifications/template_example.rst

```diff
@@ -210,12 +210,12 @@ And we want to add a text under the URL that everyone can understand:
 
 1. In the **Default translation** we add the English text
 2. Remember to **Save**
-3. **Right-clic** in "Email HTML body" field > **Source code**
+3. **Right-click** in "Email HTML body" field > **Source code**
 4. Select all the code to **copy** it
 5. Go back to the template header by clicking on the **Notification templates** value at the top
 6. In **Template translations** tab click on **Add a new translation** button
-7. Select Language = **Français**
-8. **Right-clic** in "Email HTML body" field > **Source code**
+7. Select Language = **French**
```

Comment:
I think it's "Français" in the drop-down list?
```suggestion
7. Select Language = **Français**
```

Everything seems OK to me

Reply:
Oops, I forgot the language dropdown isn't localized.